build/Octopus.Server.Extensibility.Authentication.nuspec

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Per the changes for Data, we should be able to package this just using `DotNetPublish`

Reply:
fixed